src/main/java/com/google/sps/data/User.java

```diff
@@ -135,10 +140,38 @@ public String getEmail() {
 
   public void setBio(String userBio) {
       this.userBio = userBio;
+
+      //update map that counts every user bio word in MatchRanking
+      if(userBio != null) {
+          MatchRanking.addToAllUserWordCount(userBio);
+      }
   }
 
   public String getBio() {
       return userBio;
   }
 
+  //method for testing to directly add new bio to user
+  public void addNewBioToMapCountForTesting(String newBio) {
```

Comment:
nit: For consistency / ease of reading, we usually prefix testOnly (rather than suffix forTesting)

Reply:
done